api/src/main/java/com/example/dockercrudtorrexspring/lutris/Controllers/DependentsController.java

```diff
@@ -21,54 +21,75 @@ public DependentsController() throws NoSuchAlgorithmException {
     }
 
     @GetMapping(produces = "application/json")
-    public ResponseEntity<List<Dependent>> getAll() throws SQLException {
-        ArrayList<Dependent> dependents = this.dependentsServices.getAll();
+    public ResponseEntity<List<Dependent>> getAll() {
+        try{
+            ArrayList<Dependent> dependents = this.dependentsServices.getAll();
+            return new ResponseEntity<>(dependents, HttpStatus.OK);
+        }catch (Exception e) {
+            e.printStackTrace();
+        }return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Acredito que esse return deveria estar na linha de baixo. O msm vale para os outros casos.

 No intellij tem como vc formatar o código com control+alt+L (se n estou enganado), mas normalmente, nas IDEs possuem uma configuração de formatar todas vez que o código for salvo (que acaba sendo uma mão na roda).